docs/utilities/tags.md

```diff
@@ -11,10 +11,9 @@ For example, to add your own mod's saplings to the Vanilla sapling tag, you woul
 If `replace` is true, then all entries before the json specifying `replace` will be removed.
 See the [Vanilla wiki][tags] for a description of the base syntax.
 
-Forge provides two extensions on the Vanilla syntax:
+Forge extensions on the Vanilla syntax:
 
-* You may declare an `optional` array of the same format as the `values` array, but any values listed here that are not present will not cause the tag loading to error.
-This has been deprecated in favor of the vanilla method of specifying optional tag values.
+* The `optional` array was depreciated and has been removed in favor of the vanilla method of specifying optional tag values.
```

Comment:
This was removed back in 1.17, so apologies on missing it. But instead of mentioning it's removal, it should instead mention the correct way to declare optional tag values.

Additionally, `depreciated` is spelled incorrectly. It is `deprecated`.